sjb/config/common/test_cases/origin_built_installed_release.yml

```diff
@@ -73,12 +73,18 @@ extensions:
       title: "origin prerequisites"
       repository: "aos-cd-jobs"
       script: |-
+        if [ -f "sjb/inventory/${PULL_BASE_REF}.cfg" ]; then
+          evars="-e @sjb/inventory/${PULL_BASE_REF}.cfg"
+        else
+          evars=""
+        fi
         ansible-playbook -vv --become               \
                          --become-user root         \
                          --connection local         \
                          --inventory sjb/inventory/ \
                          -e openshift_deployment_type=origin  \
                          /usr/share/ansible/openshift-ansible/playbooks/prerequisites.yml
+                         ${evars}
```

Comment:
```
${evars:-}
```

Reply:
Also added quotes around it as without that you only get `@sjb/inventory/master.cfg` rather than `-e @sjb/inventory/master.cfg`